skeleton/template.yaml

```diff
@@ -39,7 +39,7 @@ spec:
           # SED_LLM_SERVER_START
           title: Large Language Model(LLM) Server
           description: The model server container to use
-          ui:help: If you choose vLLM, ensure your cluster has Nvidia GPU supported, and has enough CPU and memory resources
+          ui:help: If you choose vLLM, ensure your cluster has Nvidia GPU supported, and has enough CPU and memory resources. You can find detailed requirements for vllm at https://docs.vllm.ai/en/latest/getting_started/installation.html#requirements
```

Comment:
```suggestion
          ui:help: If you choose vLLM, ensure your cluster has Nvidia GPU supported, and has enough CPU and memory resources. You can find detailed requirements for vLLM at https://docs.vllm.ai/en/latest/getting_started/installation.html#requirements
```

We also need to get all UX PRs reviewed by the UX team.. I am currently tagging Masha to review this PR

Reply:
that link is from vllm docs to install on linux OS.. but our vllm image is RH built vllm `quay.io/rh-aiservices-bu/vllm-openai-ubi9:0.4.2`, and install on cluster. I'm not sure if that link is the best fit to be displayed here

Reply:
> We also need to get all UX PRs reviewed by the UX team.. I am currently tagging Masha to review this PR

As an improvement here, to avoid back & forth, I think we might need to have more specific acceptance criteria from the UX team.

Reply:
> that link is from vllm docs to install on linux OS.. but our vllm image is RH built vllm quay.io/rh-aiservices-bu/vllm-openai-ubi9:0.4.2, and install on cluster. I'm not sure if that link is the best fit to be displayed here

I put this link as to my understanding the `vllm` mentioned is the `vllm` python library right? Among the requirements is to be installed on a linux based OS. I'd guess that this image quay.io/rh-aiservices-bu/vllm-openai-ubi9:0.4.2 is based on linux OS too.

Reply:
no. the vllm mentioned here means the vllm model server image going to be pulled in and deployed on cluster. which is `quay.io/rh-aiservices-bu/vllm-openai-ubi9:0.4.2`. 
and since this image is already built with vllm installed. I don't know if the user needs to know about how the pre-built image been built. 

For this message here, it's asking user to check if their cluster has enough memory and CPU on cluster, as well as if able to tolerate a nividia GPU for the model server image to be deployed & run.

Reply:
Oh ok that makes it clearer then.

> I don't know if the user needs to know about how the pre-built image been built.

Yes I agree, they shouldn't get to know that, at least from this help text.

> For this message here, it's asking user to check if their cluster has enough memory and CPU on cluster, as well as if able to tolerate a nividia GPU for the model server image to be deployed & run.

Yeah, I'll try to edit and include more specified info around that.

Reply:
@thepetk I have updated the Approach and Acceptance Requirement to be bit more specific, please check if it helps.

Reply:
@yangcao77 @maysunfaisal I've been looking around to check what info we could include here. My two cents.

We mention three things in the initial message: (1) Nvidia GPU support, (2) ensure memory resources, (3) ensure CPU resources are ok.

* From the link I've shared above, I'd keep the `GPU: compute capability 7.0 or higher (e.g., V100, T4, RTX20xx, A100, L4, H100, etc.)` info.
* Memory resources wise, the only info specific info we could include is the image size but I'm not sure how much sense it makes. More detailed, I believe the required memory size is influenced by the model & its parameters. That said I'd say is better if we'd like to be more specific.
* Now for the CPU I guess it also plays role how many & what type of requests the vLLM server receives so again I'm not sure how we can calculate a minimum requirement over that.

That said an alternative suggestion to what I shared above could be:

```
If you choose vLLM, ensure your cluster has Nvidia GPU supported (with compute capability 7.0 or higher) and has enough CPU & memory resources for the model you'd like to work with.
```
 
 WDYT?

Reply:
@MariaLeonova what do you think of the above suggestion?

Reply:
So the user will have an option to copy the link and review more detailed requirements?

Reply:
According to the discussion above, we won't use the link as it's mentioning the requirements of the `vLLM` python library instead of the whole server.

Reply:
Adding a screenshot too to show how it looks.
![image](https://github.com/user-attachments/assets/0e845c75-7d26-4f41-bb39-79f14e4f5d91)

Reply:
@thepetk small nit: can we put a full stop after the first, top sentence? And, I think, it should say "`ensure that...`".
Otherwise looks good!

Reply:
@MariaLeonova let me know if the following is ok and I'll update the PR:

```
If you choose vLLM, ensure that your cluster has Nvidia GPU supported (with compute capability 7.0 or higher). Also, it has enough CPU & memory resources for the model you'd like to work with.
```

Reply:
Can we say, `"also that it has"` or `"also it should have"`?

Reply:
Updated! Feel free to review :)